src/Illuminate/Routing/Route.php

```diff
@@ -997,6 +997,7 @@ public function getMissing()
         return is_string($missing) &&
             Str::startsWith($missing, [
                 'O:47:"Laravel\\SerializableClosure\\SerializableClosure',
+                'O:55:"Laravel\\SerializableClosure\\UnsignedSerializableClosure',
```

Comment:
This seems duplicated in `RouteAction`. Could prob. be consolidated

Reply:
Indeed. Going to keep it like this for now.